src/pages/dar_application/ResearcherInfo.js

```diff
@@ -150,35 +157,115 @@ export default function ResearcherInfo(props) {
         ]),
 
         div({ className: 'form-group' }, [
-          div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group' }, [
-            label({ className: 'control-label rp-title-question' }, [
-              '1.3 Principal Investigator* ',
-              span({}, ['I certify that the principal investigator listed below is aware of this study'])
+          div( {className: 'row no-margin' }, [
+            div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group' }, [
+              label({ className: 'control-label rp-title-question' }, [
+                '1.3 Principal Investigator* ',
+                span({}, ['I certify that the principal investigator listed below is aware of this study'])
+              ])
+            ]),
+            div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group' }, [
+              input({
+                type: 'text',
+                name: 'investigator',
+                id: 'inputInvestigator',
+                value: investigator,
+                disabled: true,
+                className: invalidInvestigator && showValidationMessages ? 'form-control required-field-error' : 'form-control',
+                required: true
+              }),
+              span({
+                className: 'cancel-color required-field-error-span', isRendered: (invalidInvestigator) && (showValidationMessages)
+              }, ['Required field'])
+            ])
+          ]) 
+        ]),
+
+        div({className: 'form-group'}, [
+          div({className: 'row no-margin'}, [
+            div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
+              label({className: "control-label rp-title-question"}, [
+                '1.4 Internal Lab Staff',
+                //NOTE: format this line?
```

Comment:
Is this something that needs to be resolved?

Reply:
Nope, it's an old comment that I forgot to delete